src/compiler.adoc

```diff
@@ -1019,8 +1019,8 @@ Ran 1 tests containing 1 assertions.
 1 failures, 0 errors.
 ----
 
-You can observe that expected assert failure is successfully printed in the console. To
-fix the test, just change the `=` with `not=` and run again the file:
+You can observe that the expected assert failure is successfully printed in the console. To
+fix the test, just change the `=` with `not=` and run the file again:
```

Comment:
- article: just inserting "the"
- grammar: run again the file -> run the file again